policyengine_us/parameters/gov/states/nd/tax/income/credits/marriage_penalty/taxable_income_base.yaml

```diff
@@ -2,13 +2,22 @@ description: North Dakota has a nonrefundable marriage-penalty credit for joint
 values:
   2021-01-01: 12_550
   2022-01-01: 12_950
+  2023-01-01: 13_850
 
 metadata:
-  label: marriage-penalty credit taxable income base amount
-  unit: USD
+  label: North Dakota marriage-penalty credit taxable income base amount
+  unit: currency-USD
+  uprating: gov.irs.uprating
+  rounding:
+    type: downward
+    interval: 50
```

Comment:
```suggestion
  uprating:
    parameter: gov.irs.uprating
    rounding:
      type: downward
      interval: 50
```

Reply:
can we nest all of the rounding metadata like this?

Reply:
We can't find any legal references that support this, they only appeared in the instruction,  the only reason we round it to the nearest 50 is because the previous years are all multiple of 50s.

Reply:
The comment is more about the format of the uprating and not the referencing - even though we need to investigate that as well

Reply:
does the law not talk about the "cost-of-living-adjustment"? However the rounding might originate in a different legal code section

Reply:
Thank you, it does mention cost-of-living adjustment. so it follows IRS standards. 
![image](https://github.com/PolicyEngine/policyengine-us/assets/113824411/98e5c7ed-5ac4-4de2-9498-766d23211832)